framework/include/outputs/TableOutput.h

```diff
@@ -96,3 +100,47 @@ class TableOutput : public AdvancedOutput
   /// Enable/disable output of time column for Postprocessors
   const bool _time_column;
 };
+
+template <typename T>
+void
+TableOutput::outputReporter(const ReporterName & name)
```

Comment:
I am thinking that this should be moved to the source file and have explicit instantiation of the supported types, this way you will see compiler options if something is added without support by child objects.

Reply:
Good idea